src/config-projects/apache.ts

```diff
@@ -29,6 +29,7 @@ export const config: RepoConfig = {
     '21232173020@chatroom',
     '19502430226@chatroom',
   ],
+  'apache/incubator-pegasus' : '21353174418@chatroom',
```

Comment:
It requires entry to be in alpha-beta order. So you need to put `apache/incubator-pegasus` in front of `apache/incubator-shenyu`.